packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/resources/TransformResource.java

```diff
@@ -32,6 +37,42 @@ public Response mathML2ACSet(List<String> list) {
 		return skemaProxy.convertMathML2ACSet(list);
 	}
 
+	@POST
+	@Path("/mathml-to-amr/{framework}")
+	@APIResponses({
+		@APIResponse(responseCode = "500", description = "An error occurred retrieving the AMR"),
+		@APIResponse(responseCode = "400", description = "Query must contain a valid MathML and a Model framework")
+	})
+	public Response mathML2AMR(
+		final List<String> mathml,
+		@PathParam("framework") final String framework
+	) {
```

Comment:
It is a little bit weird that parts of the payload is on PathParam and part of the payload is in the POST-body, is there a particular reason why we want to structure the API like this? Why not just dump everything into the POST-body?

Reply:
That's what I had originally, this way it's a bit easier to duplicate the endpoints. But I guess it doesn't matter.